libclc/README.TXT

```diff
@@ -23,29 +23,41 @@ generic implementations of most library requirements, allowing the target
 to override the generic implementation at the granularity of individual
 functions.
 
-libclc currently only supports the PTX target, but support for more
+libclc currently supports NVPTX, AMDGPU and SPIRV targets, but support for more
 targets is welcome.
 
-Compiling and installing with Make
-----------------------------------
+Compiling and installing
```

Comment:
Do you think we need to mention the use of `LIBCLC_TARGETS_TO_BUILD` to control which targets are built? The default is "all".

Reply:
I didn't know there was such a variable so, yes I'll mention that.

Reply:
done.